src/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -73,27 +71,25 @@ public void ComputeData()
             }
 
             // Get the assembly info of System.Object and set it as assembly info for primitives
-            if (_assemblyFilter is DotNetFrameworkFilter)
+            var systemObjectMemberDependency = MemberDependency.FirstOrDefault(t => string.Equals(t.MemberDocId, "T:System.Object", StringComparison.Ordinal) && _assemblyFilter.IsFrameworkAssembly(t.DefinedInAssemblyIdentity));
```

Comment:
I've seen a customer in precisely that situation :) Since the filters are meant more as to calculate which ones should be included, I'm going to remove it